include/MiniLua/math.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef MINILUA_MATH_HPP
+#define MINILUA_MATH_HPP
+
+#include <climits>
+#include <cmath>
+#include <cstddef>
+#include <functional>
+#include <random>
+#include <string>
+
+#include "MiniLua/values.hpp"
+
+namespace minilua::math {
+const long MAXINTEGER = LONG_MAX;
+const long MININTEGER = LONG_MIN;
+const double PI = M_PI;
+const double HUGE = HUGE_VAL;
+
+// needs to be in the hpp so randomseed is testable.
+// the only other way would be a getter for this variable
+std::default_random_engine random_seed; // NOLINT
```

Comment:
This has to be defined in a cpp otherwise there (very likely) might be multiple definitions of this variable in the binary.

(see the error in the AddressSanitizer test)